### ISSUE TITLE
Using ui-route ( angular-meteor) warning is shown on the page

### ISSUE THREAD

**Issue by sauliuni** [gterrono/houston#365]
Hello, 

if im using angularui:angular-ui-router im getting following error in the bottom of the page:

```
Oops, looks like there's no route on the client or the server for url: "http://localhost:3000/."
```

Navigating to admin works, and houston is loaded.

**Comment by Menda** [comment]
Yes, I also get this error using FlowRouter as my main router. I get this "Oops" error in every address outside the `/admin`. Any solution?

**Comment by mornir** [comment]
http://stackoverflow.com/questions/31035196/iron-router-conflict-with-angular-ui-router

Personally, I just stopped using Houston. It is too much hassle to implement iron:router only to get rid of that warning.
